kalite/contentload/management/commands/unpack_assessment_zip.py

```diff
@@ -51,19 +51,27 @@ def handle(self, *args, **kwargs):
             return
 
         if is_valid_url(ziplocation):  # url; download the zip
-            logging.warn("Downloading assessment item data from a remote server. Please be patient; this file is big, so this may take some time...")
+            print "Downloading assessment item data from a remote server. Please be patient; this file is big, so this may take some time..."
             # this way we can download stuff larger than the device's RAM
             r = requests.get(ziplocation, prefetch=False)
+            content_length = r.headers.get("Content-Length")
+            print "Downloaded size: ", str(int(content_length) // 1024 // 1024) + " MB" if content_length else "Unknown"
```

Comment:
Thanks for noticing :) Well, Django already has one in templatetags... I didn't bother because it's a pretty simple line that I don't see breaking... rather it's more or less problematic to download a zip with undefined contents of that size from a static location, even to add another redundant function to fle_utils which I rather see as a package on death row because it's attracting such a lot of mess and wheel reinventions.